docs/pages/_app.js

```diff
@@ -142,6 +146,61 @@ function AppWrapper(props) {
     }
   }, []);
 
+  const productIdentifier = React.useMemo(() => {
+    const languagePrefix = pageProps.userLanguage === 'en' ? '' : `/${pageProps.userLanguage}`;
+
+    if (product === 'material-ui') {
+      return {
+        name: 'Material UI',
+        metadata: 'MUI Core',
+        versions: [
+          { text: `v${materialPkgJson.version}`, current: true },
+          {
+            text: 'v4',
+            href: `https://v4.mui.com${languagePrefix}/getting-started/installation/`,
+          },
+          {
+            text: 'View all versions',
+            href: `https://mui.com${languagePrefix}/versions/`,
+          },
+        ],
+      };
+    }
+
+    if (product === 'joy-ui') {
+      return {
+        name: 'Joy UI',
+        metadata: 'MUI Core',
+        versions: [{ text: `v${joyPkgJson.version}`, current: true }],
+      };
+    }
+
+    if (product === 'system') {
+      return {
+        name: 'MUI System',
+        metadata: 'MUI Core',
+        versions: [
+          { text: `v${systemPkgJson.version}`, current: true },
+          { text: 'v4', href: `https://v4.mui.com${languagePrefix}/system/basics/` },
+          {
+            text: 'View all versions',
+            href: `https://mui.com${languagePrefix}/versions/`,
+          },
+        ],
+      };
+    }
+
+    if (product === 'base') {
+      return {
+        name: 'MUI Base',
```

Comment:
https://master--material-ui.netlify.app/base/getting-started/overview/

```suggestion
        name: 'Base UI',
```

Reply:
Fixed in ec3d7dd6cbcd362736164487a9789908d5d47133. I saw it again doing a demo with a person at React Summit.